tests/system/small/test_dataframe.py

```diff
@@ -3396,10 +3398,6 @@ def test_df_value_counts(scalars_dfs, subset, normalize, ascending, dropna):
         subset, normalize=normalize, ascending=ascending, dropna=dropna
     )
 
-    # Older pandas version may not have these values, bigframes tries to emulate 2.0+
```

Comment:
Let's remove this from test_series as well https://github.com/search?q=repo%3Agoogleapis%2Fpython-bigquery-dataframes%20%22Older%20pandas%20version%20may%20not%20have%20these%20values%22&type=code

Reply:
done

Reply:
Thanks, just one one more test `test_value_counts_w_cut` in test_series.py

Reply:
done